packages/component-library/src/CardList/CardList.js

```diff
@@ -77,260 +91,216 @@ const useStyles = makeStyles(theme => ({
+                  <ListItem key={shortid.generate()} className={classes.nested}>
+                    <Checkbox
+                      value={activeTags[topic]}
+                      label={topic}
+                      onChange={() => {
+                        if (showAllStories) setShowAllStories(false);
+                        setActiveTags({
+                          ...activeTags,
+                          [topic]: !activeTags[topic]
+                        });
+                      }}
+                    />
+                  </ListItem>
+                ))}
+              </List>
+            </Collapse>
+          </>
+        ))}
       </List>
     </div>
   );
 
+  const filterCardsBasedOnActiveTags = entryTags => {
+    if (!entryTags) return false;
+    if (showAllStories) return true;
+    if (noFiltersSelected()) return true;
+    for (let i = 0; i < entryTags.length; i += 1) {
+      if (activeTags[entryTags[i]]) return true;
+    }
+    return false;
+  };
+
   return (
-    <div className={classes.root}>
-      <CssBaseline />
-      <AppBar position="fixed" className={classes.appBar}>
-        <Toolbar>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            edge="start"
-            onClick={handleDrawerToggle}
-            className={classes.menuButton}
-          >
-            <MenuIcon />
-          </IconButton>
-          <Typography variant="h6" noWrap>
-            Responsive drawer
-          </Typography>
-        </Toolbar>
-      </AppBar>
-      <nav className={classes.drawer} aria-label="mailbox folders">
-        {/* The implementation can be swapped with js to avoid SEO duplication of links. */}
-        <Hidden smUp implementation="css">
-          <Drawer
-            variant="temporary"
-            anchor={theme.direction === "rtl" ? "right" : "left"}
-            open={mobileOpen}
-            onClose={handleDrawerToggle}
-            classes={{
-              paper: classes.drawerPaper
-            }}
-            ModalProps={{
-              keepMounted: true // Better open performance on mobile.
-            }}
-          >
-            {drawer}
-          </Drawer>
-        </Hidden>
-        <Hidden xsDown implementation="css">
-          <Drawer
-            classes={{
-              paper: classes.drawerPaper
-            }}
-            variant="permanent"
-            open
-          >
-            {drawer}
-          </Drawer>
-        </Hidden>
-      </nav>
-      <main className={classes.content}>
-        <h1>Did you know?</h1>
-        <div>
+    <ThemeProvider theme={MaterialTheme}>
+      <div className={classes.root}>
+        <CssBaseline />
+        <AppBar position="fixed" className={classes.appBar}>
+          <Header title="Story Cards" />
+        </AppBar>
+        <nav className={classes.drawer} aria-label="mailbox folders">
+          <Hidden smUp implementation="css">
+            <Drawer
```

Comment:
The Drawer should be behind the AppBar
<img width="769" alt="Screen Shot 2019-09-08 at 5 06 41 PM" src="https://user-images.githubusercontent.com/10056079/64496561-0d639e00-d25b-11e9-8757-60e4782cc631.png">

Reply:
Never mind, we'll need to redo this anyways